### PR TITLE
update CHANGES (master / 1.1.0b4)

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
please proofread.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -135,66 +135,66 @@ Compatibility notes:
 
 - init: the --encryption argument is mandatory now (and there are several choices).
 - moved "borg migrate-to-repokey" to "borg key migrate-to-repokey".
-- "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
+  "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
```
Comment (ThomasWaldmann) [inline docs/changes.rst]: this does not seem related to the previous entry, so the "-" should stay there.

Comment (enkore) [inline docs/changes.rst]: I saw it as one change, introducing the new name and deprecating the old one in the same move

Comment (enkore) [inline docs/changes.rst]: btw. those that I moved around I moved up/down for the impact I see for them, that's why I put the network, timestamp and hashindex fixes at the top of the bug fixes

**Comment by ThomasWaldmann** [comment]
I'll leave this PR open until the last other b4 changes / PRs are merged and this has been updated accordingly.

**Comment by ThomasWaldmann** [comment]
@textshell if you have time, have a look.

**Comment by ThomasWaldmann** [comment]
ok, just added latest changes. will collapse / merge soon.
